FormalConjectures/ErdosProblems/677.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 677
+*Reference:* [erdosproblems.com/677](https://www.erdosproblems.com/677)
+-/
+
+namespace Erdos677
+
+/-
+We use the same definition as is Erdős Problem 678.
+-/
+
+/--
+Write $M(n, k)$ be the least common multiple of ${n+1, \dotsc, n+k}$, denoted here as
+`lcmInterval n k`.
+-/
+def lcmInterval (n k : ℕ) : ℕ := (Finset.Ioc n (n + k)).lcm id
```

Comment:
Since you are reusing a definition from `678.lean` I suggest you create a new file in `ForMathlib` containing this definition. Possibly it could be put in a new file `Data/Nat/Interval.lean`. A bonus (but optional) would be if you wanted to try to find the minimal assumptions so that this definition makes sense (it should make sense outside just the naturals). For example `Finset.lcm` that you are using is defined in any `NormalizedGCDMonoid`.